src/components/electron/Filestream.js

```diff
@@ -19,13 +19,20 @@ class Filestream extends React.Component {
     super(props)
     this.state = {
       original: {},
-      intervalId: null
+      prevFileInstance: null
     }
   }
 
   componentDidUpdate(prevProps) {
     if (prevProps.file !== this.props.file) {
-      tail(this.props.file[0]).start(
+      const Filetail = new Tail(this.props.file[0])
```

Comment:
You don't need a `new` here. The util is just a function that returns an object and all the state it needs is in its closure (it doesn't rely on `this` for context).

Reply:
fixed in https://github.com/rowolff/babel-poe/commit/8d18bd30cf24aa5442c1e14cf46ec446c10b41b3